lib/tasks/brakeman.rake

```diff
@@ -3,11 +3,13 @@ namespace :brakeman do
   task :run do
     require "brakeman"
 
-    Brakeman.run(
+    result = Brakeman.run(
       app_path: ".",
       quiet: true,
       pager: false,
       print_report: true
     )
+
+    exit Brakeman::Warnings_Found_Exit_Code unless result.filtered_warnings.empty?
```

Comment:
Would this also exit if there brakeman errors?

Reply:
Ah, right you are. I copied this out of the (now deprecated and removed) Brakeman Rake task, which didn't account for errors for some reason. I've fixed this now